spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceHandlerInterceptor.java

```diff
@@ -62,26 +62,19 @@ public TraceHandlerInterceptor(BeanFactory beanFactory) {
 	@Override
 	public boolean preHandle(HttpServletRequest request, HttpServletResponse response,
 			Object handler) throws Exception {
-		if (isErrorControllerRelated(request)) {
-			if (log.isDebugEnabled()) {
-				log.debug("Skipping creation of a span for error controller processing");
-			}
-			return true;
-		}
-		if (isSpanContinued(request)) {
-			if (log.isDebugEnabled()) {
-				log.debug("Skipping creation of a span since the span is continued");
-			}
-			return true;
-		}
 		String spanName = spanName(handler);
-		Span span = getTracer().createSpan(spanName);
+		boolean reuseSpan = getTracer().isTracing();
```

Comment:
I think we use the terms join or continue? which are we using more often? maybe do that vs a 3rd way (reuse)